templates/scenarios/sso-tab-js/teamsfx/app.yml

```diff
@@ -31,7 +31,7 @@ deploy:
   - uses: azureStorage/deploy # Deploy bits to Azure Storage Static Website
     with:
       workingDirectory: .
-      distributionPath: . # Deploy base folder
+      distributionPath: ./build # Deploy base folder
```

Comment:
We still need to deploy all the project files, like packages.json, to run the project

Reply:
This seems to work, if I use `.`, the index.html is not found, do we need to change the tab url to `https://domain/build/index.html`?

Reply:
I think `./build` is correct for tab.